java/client/src/main/java/glide/api/models/commands/stream/StreamGroupOptions.java

```diff
@@ -15,11 +15,11 @@
 @Builder
 public final class StreamGroupOptions {
 
-    // Redis API String argument for makeStream
-    public static final String MAKE_STREAM_REDIS_API = "MKSTREAM";
+    // Valkey API String argument for makeStream
```

Comment:
It is ok to have this change, but we have another task to update all references. Don't worry about redis links/refs.